### ISSUE TITLE
Installation failing on Ubuntu Server 17.10 "Artful Aardvark"

### ISSUE THREAD

**Issue by brianwarner** [openhab/openhabian#379]
I ran across an issue installing OpenHABian on Ubuntu 17.10.  It doesn't appear to be an OpenHABian issue directly, but I wanted to log it here because it caused my installer to fail and it may be useful to others.

When installing Node.js, it appears the installer queries the node release servers and looks for a matching release rname.  At this time, only the Trusty, Xenial, Yakkety, and Zesty Ubuntu releases are supported.

I believe this is an upstream issue.  I don't think there's anything to be done directly by OpenHABian (aside from wait for Artful to be supported, and for me to use a different distro for this install).

**Comment by ThomDietrich** [comment]
Hey!
Interesting. As you've already realized, we are utilizing the nodesource.com provided installer script, nothing to fix on our side... I wonder however if the error might be fixed by a newer nodejs core version installer script. The routine currently installs v7 but v8 and even v9 are available.
NodeRED also seems to recommend Node.JS v8.
Would you like to test and create a PR?

* https://github.com/openhab/openhabian/blob/master/functions/nodejs-apps.sh#L18
* https://nodejs.org/en/download/package-manager/#debian-and-ubuntu-based-linux-distributions
* https://nodered.org/docs/getting-started/installation

**Comment by brianwarner** [comment]
Hi Thom,
Good call, I just tweaked the setup script and changing to v8 worked.  It installed successfully, and I eventually was able to get fronttail up and running via `openhabian-setup.sh`.  I see also they also have pre-release support in place for Ubuntu 18.04 LTS (Bionic): https://deb.nodesource.com/node_8.x/dists/

Is there any other functionality I should test prior to creating a PR? It looks like the changes from v7 to v8 were mostly additions, and there's the added benefit that v8 is an LTS release of nodejs itself: https://nodejs.org/en/blog/release/v8.0.0/
